javascript/javascript1/week2/formal-fullname.js

```diff
@@ -0,0 +1,49 @@
+// In order to add this optional formal title before fullname, I write another function
+// i use "!" not logical operator to check if name & surname part is empty
+
+function getFullname(firstname, surname, useFormalName) {
+  if (!firstname && !surname) {
```

Comment:
Should this be
```suggestion
  if (!firstname || !surname) {
```
if one of the names is missing? Else the name `getFullname` would be misleading.